pkg/env/node_scan.go

```diff
@@ -3,6 +3,16 @@ package env
 import "time"
 
 var (
-	// NodeRescanInterval will set the duration for when to fetch node inventory to be scanned for vulnerabilities
-	NodeRescanInterval = registerDurationSetting("ROX_NODE_RESCAN_INTERVAL", 4*time.Hour)
+	// NodeScanningInterval is the base value of the interval duration between node scans.
+	NodeScanningInterval = registerDurationSetting("ROX_NODE_SCANNING_INTERVAL", 4*time.Hour)
+
+	// NodeScanningIntervalDeviation is the percentage by which each interval
+	// duration between node scans will deviate from the base interval time. A value
+	// equal or lower than zero disables deviation.
+	NodeScanningIntervalDeviation = RegisterIntegerSetting("ROX_NODE_SCANNING_INTERVAL_DEVIATION", 10)
```

Comment:
That's a nice suggestion. It definitely makes things easier. :+1: 

Addressing this will also cover https://github.com/stackrox/stackrox/pull/5087#discussion_r1126155520 and https://github.com/stackrox/stackrox/pull/5087#discussion_r1126145454